### PR TITLE
Initial content for privacy fundamentals

### PR DESCRIPTION
**Pull request by omri374**
This PR proposes an initial version for the privacy engineering fundamentals content. It has two main parts:
1. Data handling guidelines
2. Privacy frameworks

This content has been reviewed internally by legal and compliance functions and has been approved for publishing.

- [X] Link to the issue you are solving (so it gets closed)
- [X] Label the pull request with the appropriate area(s)
- [X] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [X] Assign the project - Engineering Playbook Backlog
- [X] Assign the pull request to yourself

### CODE REVIEW

File: docs/privacy/data-handling.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+when this is put in practice we find that we only maintain data that is adequate,
+relevant and limited to what is necessary in relation to the purposes for which they are processed.
+This is particularly important for personal data. Once you have personal data there are many rules and regulations that apply,
+some examples of these might be HIPPA, GDPR, CCPA.
+The customer should be aware of and surface any applicable regulations that apply to their data.
+Furthermore the [seven principles of privacy by design](http://dataprotection.industries/wp-content/uploads/2017/10/privacy-by-design.pdf)
+should be reviewed and considered when handling any type of sensitive data.
+Additional information:
+
+- [Microsoft Trust Center](https://www.microsoft.com/en-us/trust-center/privacy)
```
Comment (biancafbmd) [inline docs/privacy/data-handling.md:105]: Are these last bullet points are a subset of the last question? If so, maybe adding some tabs would improve readability here?

File: docs/privacy/data-handling.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+If there is a need to access production data,
+CSE needs to review the need with their lead and work with the customer to put audits in place verifying what data was accessed.
+
+Production data must only be shared with approved members of the engagement team and must not be processed/transferred outside of the customer controlled environment.
+
+Customers should provide CSE with a copy of the requested data in a location managed by the customer.
+The customer should consider turning any logging capabilities on so they can clearly identify who has access and what they do with that access.
+CSE should notify the customer when they are done with the data and suggest the customer destroy copies of the data if they are no longer needed.
+
+Our guiding principles when handling data in an engagement include:
```
Comment (biancafbmd) [inline docs/privacy/data-handling.md:70]: Suggestion to make this a sub-header to break the content a bit to improve readability.

File: docs/privacy/data-handling.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+- Only provide access to production data when required and explicitly state 5Ws behind the data access.
+- Limit access to the data set required to meet the engagement objectives.
+- Explicitly state the intended purpose of data that can be used for engagement.
+- Only share production data with the approved members of the engagement team.
+- The entire team should work together to ensure that there are no dead copies of data. When the data is no longer needed,
+the team should promptly work to clean up engagement copies of data.
+- Do not send any production data outside the customer-controlled environment.
+
+Questions to consider when working with data:
```
Comment (biancafbmd) [inline docs/privacy/data-handling.md:80]: Same suggestion to add a sub-header here

File: docs/privacy/privacy-frameworks.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+1. [Obtaining non-sensitive data](#obtaining-non-sensitive-data)
+2. [Establishing trusted research and modeling environments](#trusted-research-and-modeling-environments)
+3. [Creating privacy preserving data and ML pipelines](#privacy-preserving-data-pipelines-and-ml)
+4. [Data loss prevention](#data-loss-prevention)
+
+### Obtaining non-sensitive data
+
+In many scenarios, analysts, researchers and data scientists require access to a non-sensitive version or sample of the private data.
+In this section we focus on two approaches for obtaining non-sensitive data.
+Note that these two approaches do not guarantee that the outcome would not include private data, and additional measures should be applied.
```
Comment (biancafbmd) [inline docs/privacy/privacy-frameworks.md:32]:
```suggestion

**Note:** These two approaches do not guarantee that the outcome would not include private data, and additional measures should be applied.
```

**Review by biancafbmd — APPROVED** [review]
Great addition to the playbook :)

File: docs/privacy/data-handling.md (hunk 2 above)
Comment (omri374) [reply docs/privacy/data-handling.md:70]: Done

File: docs/privacy/data-handling.md (hunk 3 above)
Comment (omri374) [reply docs/privacy/data-handling.md:80]: Done

File: docs/privacy/data-handling.md (hunk 1 above)
Comment (omri374) [reply docs/privacy/data-handling.md]: Agreed. Done

**Review by TessFerrandez — APPROVED** [review]
LGTM!

**Pull request merged**